spec/praxis/trait_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
+
+describe Praxis::Trait do
+
+  subject(:trait) do
+    Praxis::Trait.new do
+      description 'my awesome trait'
+
+      routing do
+        prefix '/:app_name'
+      end
+
+      response :something
+      response :nothing
+
+      params do
+        attribute :app_name, String
+        attribute :order, String,
+          description: "Field to sort by."
+      end
+
+    end
+  end
+
+  context 'describe' do
+    subject(:describe) { trait.describe }
+
+    its([:description]) { should eq('my awesome trait') }
+    its([:responses]) { should eq [[:something,{}], [:nothing,{}]] }
```

Comment:
for responses: it feels a bit weird of a check to have to know that each response is a tuple of `name` and `something_else` (which happens to be response options...)
It might be nicer to have that as a hash internally, which would show up as a hash here...which makes it a little more palatable I guess (yes, you'd still might wonder what the "values" of the hash are...but at least you can do `[:responses].keys` ... to see all the names...